contentcuration/contentcuration/frontend/shared/constants.js

```diff
@@ -63,6 +63,8 @@ export const MAX_FILE_SIZE = 209715200;
 
 export const ASPECT_RATIO = 16 / 9;
 
+export const THUMBNAIL_WIDTH = 250;
```

Comment:
No need to do this now, but probably good to set this as the maxwidth for thumbnails across the platform

Reply:
In terms of rendering?

Reply:
Yes, I've sort of just been adding max widths to make the styling/responsiveness work ([example](https://github.com/learningequality/studio/blob/develop/contentcuration/contentcuration/frontend/channelEdit/components/edit/DetailsTabView.vue#L108)), but it seems nice to make this more of a set thing on the thumbnails themselves

Reply:
OK - I added it everywhere that we upload thumbnails to ensure that we use that in Croppa, but yeah a follow up on the rendering would be good.

The other thing is that this does not need to be the final value - but good to have it encoded in one place so we can change it with a one liner!